llmcode/relationship_mapper.py

```
@@ -0,0 +1,576 @@
+                        relationship_type=RelationshipType.CONTAINS,
+                        context=f"{entity.parent} contains {entity.name}",
+                        line_number=entity.line_number
+                    )
+                    self.relationships.append(relationship)
+                    
+    def _find_entity_by_name(self, name: str, file_path: str, entity_type: Optional[str] = None) -> Optional[str]:
+        """Find an entity by name and file path"""
+        for entity_key, entity in self.static_analyzer.entities.items():
+            if (entity.name == name and 
+                entity.file_path == file_path and 
+                (entity_type is None or entity.type == entity_type)):
```

Flake8 (7.2.0)

[error] visually indented line with same indent as next logical line

(E129)